index.js

```diff
@@ -356,6 +356,19 @@
     ($.Undefined  = type0('Undefined'))
   ];
 
+  // Enum :: [Any] -> Type
+  $.Enum = function(constants) {
+    return {
+      type: 'sanctuary-def/Enum',
+      test: function(x) {
+        return $.Array(Any).test(constants) && constants.indexOf(x) !== -1;
```

Comment:
Yeah, I was more just trying to ensure that the value I'm checking has an `indexOf` method. Should I just use:

```js
typeof constants.indexOf === 'function'
```